recipe/meta.yaml

```diff
@@ -14,20 +14,20 @@ source:
         - skip_failing_test.patch  # [osx]
 
 build:
-    number: 3
+    number: 4
     skip: True  # [win]
 
 requirements:
     build:
-        - sqlite
+        - sqlite 3.9*
```

Comment:
Part of the reason I ask is with `python` in mind.

After looking up the [ABI compatibility]( http://abi-laboratory.pro/tracker/timeline/sqlite/ ) for `sqlite`, I have to say I'm a bit surprised this causes issues. Is it possible there is something more going on?